app/controllers/welcome_controller.rb

```diff
@@ -2,7 +2,7 @@
 
 # contains all of the methods for the welcome page
 class WelcomeController < ApplicationController
-  def login; end
-
-  def index; end
+  def index
+    redirect_to homepage_index_path if params[:username] == 'Dogwalking' && params[:password] == 'pawsome'
```

Comment:
but what about catsitting jobs? Surely they would need to use the password "purrfect"

(sorry, I couldn't resist.... but you started it)

Reply:
Aha! You have overlooked an important detail. Cats also have paws!

Reply:
I guess that's quite fur-tunate.
Cat-astrophe avoided.